testing

```diff
@@ -0,0 +1 @@
+<p>Trying to test this out</p>
```

Comment:
If I merged this right now, the testing file would end up in the Wilde Trials repository and would need to be removed before updating the live site.

Reply:
I am just wondering if I have followed the right steps: creating a file; making a commit; sending a pull request; updating my desktop clone. If what I have done looks right, then I will delete the testing file.

Reply:
After sending a pull request, you need to wait until it is reviewed and merged before updating your desktop clone. Otherwise that's right.